<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="buildButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>260</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Line3">
    <item>
     <widget class="QLabel" name="ratioLabel">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Ratio:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="ratioSlider">
      <property name="minimumSize">
       <size>
        <width>192</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="suffleButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Line2">
    <item>
     <widget class="QLabel" name="offsetLabel">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Offset:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="offsetEdit">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="tangentLabel">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Tangent:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="tangentBox">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Linear</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cosine</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>260</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="line1">
    <item>
     <widget class="QLabel" name="numLabel">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="numEdit">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="stackLabel">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Stack:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stackEdit">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
